README.md

```diff
@@ -18,36 +16,40 @@ Planned features & enhancements are:
 
 Please join our mailing list to discuss questions regarding the project: https://groups.google.com/forum/#!forum/open-event
 
-Our chat channel is on gitter here:
+Our chat channel is on gitter [here](https://gitter.im/fossasia/open-event-android)
 
 ## Screenshots
 
 
 ## Development
 
-A native Android app using  Java  for writing code. 
+A native Android app using Kotlin for writing code.
 
 ### Android App Development Set up
 
-Please find info about the set up of the Android app in your development environment [here](docs/Android_App_Setup.md).
+Please find info about the set up of the Android app in your development environment.
+
+### Sample App
+
+A sample app for testing purposes is automatically built after commits in the repository. Please download and test the app that is using the code from the dev branch [here](https://github.com/fossasia/open-event-android/blob/apk/app-release-unsigned.apk) and the code from the master branch [here](https://github.com/fossasia/open-event-android/blob/apk/open-event-master-app-release-unsigned.apk).
```

Comment:
The information is already there about the apk branch.add this info there.
Don't use specific URLs, these might change just add the link to the apk branch

Reply:
Added apk branch link for now. PR Updated